app/src/main/java/com/ivanpodlesnykh/playlistmaker/SearchActivity.kt

```diff
@@ -43,6 +49,10 @@ class SearchActivity : AppCompatActivity() {
         .build()
     private val MusicService = retrofit.create(MusicApi::class.java)
 
+    val searchRunnable = Runnable { searchRequest() }
```

Comment:
```suggestion
    private val searchRunnable = Runnable { searchRequest() }
```
Можно сделать приватной